components/images/images.css

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+@import url("../../styles/doc.css");
+
+
+
+.btn-wrapper{
+    margin-top: 3rem;
+}
+/* FOR IMAGES */
+
+.responsive-image{
+    max-width: 60%;
```

Comment:
should it always take 60% of the parent container?

Reply:
corrected, I made it relative to a diiferent container now  made some changes. Thanks